administrator/components/com_installer/src/Model/UpdateModel.php

```diff
@@ -447,12 +447,16 @@ private function install($update)
 
 		// Check the package
 		$check = InstallerHelper::isChecksumValid($package['packagefile'], $update);
-
-		// The validation was not successful. Just a warning for now.
-		// TODO: In Joomla 4 this will abort the installation
+		
 		if ($check === InstallerHelper::HASH_NOT_VALIDATED)
 		{
 			$app->enqueueMessage(Text::_('COM_INSTALLER_INSTALL_CHECKSUM_WRONG'), 'error');
+			return false;
```

Comment:
```suggestion

			return false;
```
PHPCS empty line before return statement.